### DISCUSSION TITLE
Why are my statistics beginning at 2AM?

### DISCUSSION THREAD

**Discussion by CrowMash** [teslamate-org/teslamate#2831] [Q&A] [answered]
Hi,

I just wanted to know if I made a mistake configuring TeslaMate. I live in France (UTC+1 in standard time and UTC+2 in daylight time).
In my docker-compose file I added this environment variable under teslamate : "TZ=FR"

But when I go to statistics, I can see that month, weeks or days don't start à 00:00 but 01:00 or 02:00...

![image](https://user-images.githubusercontent.com/9526631/182183017-89d3c10d-3917-4c47-b8a7-acb95f146823.png)

Do I need to do something else to have my stats begin at 00:00 ?

**Comment by DrMichael** [comment, marked as answer]
Hmmm, I have set the timezones in the docker-compose.yml:

```
  teslamate:
...
    environment:
...
      - LOCALE=${LOCALE}
      - TZ=${TM_TZ}

  database:
...
    environment:
...
      - TZ=${TM_TZ}
      - PGTZ=${TM_TZ}
      - LOCALE=${LOCALE}

  grafana:
...
    environment:
...
      - TZ=${TM_TZ}
      - PGTZ=${TM_TZ}
      - LOCALE=${LOCALE}
...

```
with .env having:
```
TM_TZ=Europe/Berlin
LOCALE=DE
LANG=de_DE.UTF-8
```

**Reply by CrowMash** [reply]
It worked perfectly, thanks a lot!
